src/message/MessageListContainer.js

```diff
@@ -105,6 +109,30 @@ class MessageListContainer extends PureComponent<Props> {
     );
   };
 
+  handleUrlLongPress = (src: string) => {
+    if (isUrlAnImage(src)) {
```

Comment:
This block contains a whole bunch of details involving the internals of the lightbox.

It's also basically duplicated from some code in `LightboxContainer`.

Let's not do that. Instead, have the lightbox code expose this functionality as something this code can invoke.

Reply:
This is definitely not a right place to handle.
`UrlLongPress` is sub part of `onLongPress`, try to embed in that. 
(may be pass a flag for differentiating which element and act accordingly)

Reply:
`onLongPress` and `onUrlLongPress` have different params, so why not use a different function by first checking the type and proceed accordingly rather than passing both params always and then checking the type?